Frontend/library/src/Util/EventEmitter.ts

```diff
@@ -581,7 +581,7 @@ export type PixelStreamingEvent =
     | PlayerCountEvent
     | WebRtcTCPRelayDetectedEvent;
 
-export class EventEmitter extends EventTarget {
+export class PixelStreamingEventEmitter extends EventTarget {
```

Comment:
We already had a class called EventEmitter, so I renamed it here. This seemed like the lesser impactful rename as opposed to renaming the one in common.